src/content/docs/en/reference/experimental-flags/serialized-configuration.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: Experimental serialized configuration
+sidebar:
+  label: Serialized configuration
+i18nReady: true
+---
+
+import Since from '~/components/Since.astro';
+
+<p>
+
+  **Type:** `boolean`<br />
+  **Default:** `false`<br />
+
+  <Since v="5.2.0" />
+</p>
+
+This feature allows access to the `astro:config` virtual module which exposes a non-exhaustive, serializable, type-safe version of the Astro configuration through two sub-modules.
+
+For a complete overview, and to give feedback on this experimental API, see the [Serialized Manifest RFC](https://github.com/withastro/roadmap/blob/feat/serialised-config/proposals/0051-serialized-manifest.md).
+
+To enable this virtual module, add the `experimental.serializeManifest` feature flag to your Astro config:
+
+```js title="astro.config.mjs" ins={3-5}
+import {defineConfig} from "astro/config"
+export default defineConfig({
+  experimental: {
+    serializeManifest: true
+  }
+})
+```
+
```

Comment:
``````suggestion

Then, configuration values can be used by any file in your Astro project:
```js title="src/utils.js" ins={1}
import { trailingSlash } from "astro:config/client";
function addForwardSlash(path) {
  if (trailingSlash === "always") {
    return path.endsWith("/") ? path : path + "/"
  } else {
    return path
  }
}
```
``````
Realized we lost the example here!